src/plugins/data_table/filter_view.js

```diff
@@ -387,15 +385,15 @@ var DataTableFilter = (function(options) {
       // decide which filter view based on type, here
       var filterMenu = null;
       switch (this.filter.type) {
-        case 'string':
-          filterMenu = new StringFilterMenu({column: options.column, parentView: this});
-          break;
-        case 'numeric':
-          filterMenu = new NumericFilterMenu({column: options.column, parentView: this});
-          break;
-        case 'list':
-          filterMenu = new ListFilterMenu({column: options.column, parentView: this});
-          break;
+      case 'string':
```

Comment:
our style guide doesn't explicitly state that cases should be indented once inside a switch statement, but the only `good` example that is used includes it.

Reply:
I think we should move towards not indenting cases inside of a switch (given that's what we do for ruby). I'm going to update our good example so that there's no indentation